src/pages/Dashboard/components/ParticipantInfoPopup/ParticipantInfoPopup.module.css

```
@@ -0,0 +1,90 @@
+  position: relative;
+}
+
+.xbutton {
+  cursor: pointer;
+  position: absolute;
+  top: 1rem;
+  right: 1rem;
+}
+
+.userName {
+  margin: 0;
```

Stylelint (17.6.0)

[error] Unexpected duplicate "margin" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)